src/plugins/timelist/Timelist.vue

```diff
@@ -377,7 +408,7 @@ export default {
                     activity.key = uuid();
                 }
 
-                activity.duration = activity.start - this.timestamp;
+                activity.duration = this.timestamp && activity.start ? activity.start - this.timestamp : undefined;
```

Comment:
See my comment above about using the `bounds.start` value as the timestamp here. Not showing the duration would probably be a bad thing - maybe talk to @charlesh88 to see if he has any thoughts on what should be displayed in this case?